platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -87,21 +87,21 @@
      */
     private static final String META_TYPE_NAME = "org.osgi.service.metatype.MetaTypeService";
 
+    /**
+     * the service factor pid.
```

Comment:
This comment doesn't seem to be in the right place.  In fact, it can probably be removed; it doesn't look like it was in the right place to begin with